components/ProgressView/BUILD

```diff
@@ -59,9 +67,13 @@ mdc_objc_library(
 mdc_objc_library(
     name = "unit_test_sources",
     testonly = 1,
+    hdrs = native.glob([
```

Comment:
This target doesn't appear to be (and shouldn't be) exporting any headers into the build environment.  Was this change made by mistake?